test_scripts/Smoke/API/017_Slider_PositiveCase.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: Slider
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [Slider] SUCCESS: getting SUCCESS:UI.Slider()
+--
+-- Description:
+-- Mobile application sends valid Slider request and gets UI.Slider "SUCCESS" response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests Slider with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if UI interface is available on HMI
+-- SDL checks if Slider is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the UI part of request with allowed parameters to HMI
+-- SDL receives UI part of response from HMI with "SUCCESS" result code
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local requestParams = {
+  numTicks = 7,
+  position = 1,
+  sliderHeader ="sliderHeader",
+  timeout = 1000
+}
+
+--[[ Local Functions ]]
+local function slider(params, self)
+  local cid = self.mobileSession1:SendRPC("Slider", params)
+  params.appID = commonSmoke.getHMIAppId()
+  EXPECT_HMICALL("UI.Slider", params)
+  :Do(function(_,data)
+      self.hmiConnection:SendNotification("UI.OnSystemContext",{ appID = params.appID, systemContext = "HMI_OBSCURED" })
+      local function sendReponse()
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {sliderPosition = 1})
+        self.hmiConnection:SendNotification("UI.OnSystemContext",{ appID = params.appID, systemContext = "MAIN" })
+      end
+      RUN_AFTER(sendReponse, 1000)
```

Comment:
@GetmanetsIrina this hardcoded delay could be replaced with `params.timeout`

Reply:
@AKalinich-Luxoft , time in RUN_AFTER is not related to timeout at all, this time is to perform some delay before sending response for emulating user action.  So rejected.